prov/opx/configure.m4

```diff
@@ -120,12 +120,25 @@ AC_DEFUN([FI_OPX_CONFIGURE],[
 		    [],
 		    [],
 		    [opx_happy=0])
-
-               AC_CHECK_DECL([HAVE_ATOMICS],
-                             [],
-                             [cc_version=`$CC --version | head -n1`
-                              AC_MSG_WARN(["$cc_version" doesn't support native atomics.  Disabling OPX provider.])
-                              opx_happy=0])
+		AS_IF([test $opx_happy -eq 1],[
+			AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+				[[#include <rdma/hfi/hfi1_user.h>]],
+				[[
+					#if HFI1_USER_SWMAJOR != 6
+					#error "incorrect version of hfi1_user.h"
+					#endif
+				]])],
+				[AC_MSG_NOTICE([hfi1_user.h version 6.x found... yes])],
+				[
+				AC_MSG_NOTICE([hfi1_user.h version 6.x found... no])
+				opx_happy=0
+				])
+		])
+		AC_CHECK_DECL([HAVE_ATOMICS],
+                     [],
+                     [cc_version=`$CC --version | head -n1`
+                      AC_MSG_WARN(["$cc_version" doesn't support native atomics.  Disabling OPX provider.])
+                      opx_happy=0])
```

Comment:
Are these lines changed accidentally?  They don't seem to be necessary or related.

Reply:
I just changed the indentation.

Reply:
After the change the indentation is still not consistent anyway :) (check L110 vs L119 vs L124 vs L138 for example)

Reply:
The old version used spaces where it shouldn't have and I left some there that I shouldn't have. I just force pushed an updated version with the proper alignment and tabs. If you really want, I can revert to the original version that doesn't line up properly and uses spaces...